hydromt_wflow/data/parameters_data.yml

```diff
@@ -12,6 +12,7 @@ corine_mapping_default:
       Sl: float32
       Swood: float32
       WaterFrac: float32
+      alpha_h1: int16
```

Comment:
It is really an integer (always 0 or 1 like a boolean) or can it be a fraction (0,7 or 0.5 etc)? I would guess float after the reprojection to wflow grid anyhow no?

Reply:
Are you sure? I think the default method to reproject is average for all and only mode for landuse. This is set in workflows.landuse.py in a dict at the top of the file. Or does the integer type means that conversion to 0 and 1 will be done after average because of the type (not sure how they convert 0.4, 0.5 and 0.6 with this). Maybe mode method might be better to use then for alpha_1 like for landuse classes?

Reply:
Yeah apparently the datatype prevented it from going to floating point values, but this does indeed feel a bit tricky to rely on (also if users don't specify the dtype correctly). I have added mode resampling for this parameter!